tests/components/litterrobot/conftest.py

```diff
@@ -119,8 +118,8 @@ async def setup_integration(
 ) -> MockConfigEntry:
     """Load a Litter-Robot platform with the provided coordinator."""
     entry = MockConfigEntry(
-        domain=litterrobot.DOMAIN,
-        data=CONFIG[litterrobot.DOMAIN],
+        domain=DOMAIN,
+        data=CONFIG[DOMAIN],
```

Comment:
Also, could it be that CONFIG is a leftover for test yaml config? Maybe you can remove that as well since we don't need that anymore

Reply:
This one never had yaml config setup. Instead, that holds the username/password data for the config entry setup.

Reply:
I get that, but is there ever a case where you use CONFIG directly? 

In the past, this was the structure for yaml based integrations and that's why we still see this pattern back

Reply:
Ahh, gotcha. If we can agree, I'll address this address this in a future change to limit the scope of this PR